api/src/database/helpers/sequence/dialects/postgres.ts

```diff
@@ -0,0 +1,13 @@
+import type { Knex } from 'knex';
+import { AutoSequenceHelper } from '../types.js';
+
+export class AutoIncrementHelperPostgres extends AutoSequenceHelper {
+	/**
+	 * Resets the auto increment sequence for a table based on the max value of the PK column.
+	 * We're assuming that the default sequence name is being used,
+	 * which is the `${tableName}_${columnName}_seq`.
+	 */
+	override resetAutoIncrementSequence(_table: string, _column: string): Knex.Raw | null {
+		return this.knex.raw(`SELECT SETVAL('${_table}_${_column}_seq', (SELECT MAX(${_column}) FROM ${_table}));`);
```

Comment:
We can fetch the sequence name to eliminate the assumption. 
Here's an example:
```sql
WITH info AS (
    SELECT
        pg_get_serial_sequence('table_name', 'pk_field') AS seq_name,
        MAX(pk_field) as max_val
    FROM
        table_name
)
SELECT
    SETVAL(seq_name, max_val)
FROM
    info;
```

Reply:
I added the snippet - it now determines the sequence names dynamically 🎉